godot-bevy-macros/src/godot_node/component.rs

```diff
@@ -5,10 +5,7 @@ use syn::punctuated::Punctuated;
 use syn::spanned::Spanned;
 use syn::{Data, DeriveInput, Meta, Token, parse_quote, parse2};
 
-struct KeyValue {
-    key: syn::Ident,
-    value: syn::Expr,
-}
+use super::attr::KeyValue;
 
 struct GodotNodeAttrArgs {
```

Comment:
Looks like this `struct GodotNodeAttrArgs` and `impl Parser` can also be moved to `attr.rs`. I'll work on that and push it to this branch.